components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/config/builder/FileBasedConfigurationBuilderTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authentication.framework.config.builder;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+public class FileBasedConfigurationBuilderTest {
+
+    private FileBasedConfigurationBuilder fileBasedConfigurationBuilder;
+
+    @Before
+    public void setUp() {
+        // Mock IdentityUtil.getIdentityConfigDirPath() to return a test path
```

Comment:
```suggestion

        // Mock IdentityUtil.getIdentityConfigDirPath() to return a test path
```
Lets add new line for other methods as well

Reply:
Resolved from https://github.com/wso2/carbon-identity-framework/pull/6222/commits/9a194170f6bf003a497ee9a6a4ee4405682ceca2